packages/cli/lib/services/parser.service.ts

```diff
@@ -117,20 +120,25 @@ class ParserService {
 
                     if (callsReferencingModelsToCheck.includes(callee.property.name)) {
                         const args = path.node.arguments as t.Expression[];
-                        const modelArg = args[args.length - 1];
-                        if (t.isStringLiteral(modelArg) && !modelNames.includes(modelArg.value)) {
-                            console.log(
-                                chalk.red(
-                                    `"${
-                                        modelArg.value
-                                    }" is not a valid model name. Please check "${filePath}:${lineNumber}". The possible model names are: ${modelNames.join(
-                                        ', '
-                                    )}`
-                                )
-                            );
-                            usedCorrectly = false;
+                        if (args.length > 1) {
+                            const models = args.slice(1);
+                            for (const model of models) {
+                                if (t.isStringLiteral(model) && !modelNames.includes(model.value)) {
+                                    console.log(
+                                        chalk.red(
+                                            `"${
+                                                model.value
+                                            }" is not a valid model name. Please check "${filePath}:${lineNumber}". The possible model names are: ${modelNames.join(
+                                                ', '
+                                            )}`
+                                        )
+                                    );
+                                    usedCorrectly = false;
+                                }
+                            }
```

Comment:
adding support for functions like `setMergingStrategy` that take multiple models as argument